src/handler.ts

```diff
@@ -5,7 +5,11 @@ const { load } = require("/var/runtime/UserFunction") as any;
 
 if (getEnvValue("DD_TRACE_ENABLED", "true").toLowerCase() === "true") {
   // tslint:disable-next-line:no-var-requires
-  require("dd-trace").init();
+  require("dd-trace").init({
```

Comment:
This only applies for the redirected handler. It won't apply if they are using the wrapper manually.

Reply:
If they use the wrapper manually, then the forwarder seems to take care of it.